server

```diff
@@ -494,7 +494,7 @@ deb_install() {
     echo_msg "Installing Scylla version $SCYLLA_VERSION for $NAME ..."
     run_cmd apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys $SCYLLA_GPG_KEY
     run_cmd mkdir -p $APT_KEYS_DIR && gpg --homedir /tmp --no-default-keyring --keyring $APT_KEYS_DIR/scylladb.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys $SCYLLA_GPG_KEY
-    run_cmd apt update
+    run_cmd apt update $APT_FLAGS
```

Comment:
`$APT_FLAGS` includes also `-qq` which AFAIR hides the output - are you aware of it? is that what we want?

Reply:
yes, i know, it used -qq be default, but we can override it with `--verbose` when running `./server`